feature: Upgrading from rocksdb-4.13 to rocksdb-6.12.6 failed initialization

**Is your feature request related to a problem? Please describe.**
<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->

Upgrading from rocksdb-4.13 to rocksdb-6.12.6 failed initialization
Error executing reinstall.sh script
```
stonedb56/install/bin/mysqld: Error opening rocks instance. status msg: Invalid argument: Default column family not specified
[ERROR] Plugin 'STONEDB' init function returned error.
[ERROR] Plugin 'STONEDB' registration as a STORAGE ENGINE failed.

```
**Describe the solution you'd like**
<!-- A clear and concise description of what you want to happen. -->

**Describe alternatives you've considered**
<!-- A clear and concise description of any alternative solutions or features you've considered. -->

**Additional context**
<!-- Add any other context or screenshots about the feature request here. -->

--- a/storage/stonedb/index/kv_store.cpp
+++ b/storage/stonedb/index/kv_store.cpp
@@ -45,12 +45,11 @@ void KVStore::Init() {
   db_option.statistics = rocksdb::CreateDBStatistics();
   // get column family names from manfest file
   rocksdb::Status status = rocksdb::DB::ListColumnFamilies(db_option, rocksdb_datadir, &cf_names);
- //if (!status.ok() && status.subcode() == rocksdb::Status::kNone) {
- if (!status.ok() && ((status.subcode() == rocksdb::Status::kNone) || (status.subcode() == rocksdb::Status::kPathNotFound)) )
- {
-    STONEDB_LOG(LogCtl_Level::INFO, "First init rocksdb, create default cloum family");
-    cf_names.push_back(DEFAULT_CF_NAME);
- }
+  if (!status.ok() && ((status.subcode() == rocksdb::Status::kNone) || (status.subcode() == rocksdb::Status::kPathNotFound)) )
+  {
+      STONEDB_LOG(LogCtl_Level::INFO, "First init rocksdb, create default cloum family");
+      cf_names.push_back(DEFAULT_CF_NAME);
+  }
 
   rocksdb::ColumnFamilyOptions rs_cf_option(options);
   rocksdb::ColumnFamilyOptions index_cf_option(options);

--- a/storage/stonedb/index/kv_store.cpp
+++ b/storage/stonedb/index/kv_store.cpp
@@ -45,10 +45,12 @@ void KVStore::Init() {
   db_option.statistics = rocksdb::CreateDBStatistics();
   // get column family names from manfest file
   rocksdb::Status status = rocksdb::DB::ListColumnFamilies(db_option, rocksdb_datadir, &cf_names);
-  if (!status.ok() && status.subcode() == rocksdb::Status::kNone) {
+ //if (!status.ok() && status.subcode() == rocksdb::Status::kNone) {
+ if (!status.ok() && ((status.subcode() == rocksdb::Status::kNone) || (status.subcode() == rocksdb::Status::kPathNotFound)) )
+ {
     STONEDB_LOG(LogCtl_Level::INFO, "First init rocksdb, create default cloum family");
     cf_names.push_back(DEFAULT_CF_NAME);
-  }
+ }
 
   rocksdb::ColumnFamilyOptions rs_cf_option(options);
   rocksdb::ColumnFamilyOptions index_cf_option(options);

